### PR TITLE
Completed assignment of week 4

### PR DESCRIPTION
**Pull request by NataMarkova**
Review my code, please

### CODE REVIEW

File: hangman.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    puts array_after_guess.join
+end
+
+puts 'Provide one word: '
+param_1 = gets.chomp
+puts "Enter only one letter: "
+one_letter = gets.chomp
+new_array = []
+new_array << one_letter
+hangman(param_1, new_array)
```
Comment (andresochoa91) [inline hangman.rb:19]: This is working good. We should be able to have more than 1 letter.

File: sums.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class Sum2
+    def initialize(a, b)
+        # a & b - instance variables
+        @a = a
+        @b = b
+        @total = a + b
+    end
+
+    def new_total
+        sum2_total = @a + @b
```
Comment (andresochoa91) [inline sums.rb]: sum2_total is kind of unnecessary in this case. We should just return `@a + @b`

File: hash_to_array.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    key_array = []
+    value_array = []
+    hash.each do |key_hash, value_hash|
+        key_array.push(key_hash)
+        value_array.push(value_hash)
+    end  
+    puts "The keys of an array: #{key_array}"
+    puts "The values of an array: #{value_array}"
+
+end
```
Comment (andresochoa91) [inline hash_to_array.rb:22]: using the methods `.keys` and `.values` would make this code shorter and more clear.

**Review by andresochoa91 — APPROVED** [review]
Great job!. Some advice right below!

File: sums.rb (hunk 2 above)
Comment (NataMarkova) [reply sums.rb]: @andresochoa91 I removed sum2_total

**Pull request merged**